examples/docs/src/components.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable react/display-name */
+
+import * as React from "react";
+import system from "system-components/emotion";
```

Comment:
This uses a lot of [system-components](https://github.com/jxnblk/styled-system/tree/master/system-components), which is shorter way of writing [styled-system](https://github.com/jxnblk/styled-system) components. Thoughts on using this in the docs site?

Reply:
Works for me. I'm a huge fan of styled-system+emotion, so happy to use system-components. I don't have anything against using rebass either.